.gitignore

```diff
@@ -15,6 +15,7 @@ cover.out
 coverage-2.txt
 coverage.txt
 dist/
+hello-astro/
```

Comment:
Do you know if this is intended? I don't think the project by default builds any such folder

Reply:
No there isn't. 
I think it provides a cleaner workflow for the local directory where one runs their command based on this recommendation https://github.com/astronomer/astro-cli/blob/main/README.md?plain=1#L155